common/python/cctrusted_base/eventlog.py

```diff
@@ -152,18 +152,23 @@ def select(self, start:int, count:int) -> None:
         self._parse()
 
         if start is not None:
-            if not 0 < start <= self._count:
+            if start == self._count:
+                LOG.info("Input start equal to count. No more event log returned.")
+                self._event_logs = []
+                return
+
+            if not 0 <= start < self._count:
                 # pylint: disable-next=line-too-long
                 LOG.error("Invalid input start. Start must be number larger than 0 and smaller than total event log count.")
-                raise ValueError('Invalid parameter start.')
+                raise ValueError("Invalid parameter start.")
 
-            self._event_logs = self._event_logs[start-1:]
+            self._event_logs = self._event_logs[start:]
 
         if count is not None:
             if not 0 < count <= len(self._event_logs):
```

Comment:
two conditions need further consideration:

- start+count should be compared against the total number of eventlogs
- if start+count is bigger than total number of eventlogs, this count input should be allowed and we should return eventlogs to the end of current list